src/plugins/saved_objects_tagging_oss/public/api.ts

```diff
@@ -126,26 +126,42 @@ export interface SavedObjectsTaggingApiUi {
    * Parse given query using EUI's `Query` syntax, and return the search term and the tag references
    * to be used when using the `_find` API to retrieve the filtered objects.
    *
+   * @remark if the query cannot be parsed, `searchTerm` will contain the raw query value, and `valid`
```

Comment:
comment: I hope we plan to show the raw query and let the end user know why the query is invalid in the UI so that they can fix it.